### ISSUE TITLE
Mention requirements (e.g., git) earlier in installation notes?

### ISSUE THREAD

**Issue by bratslavia** [junegunn/vim-plug#1091]
This may be obvious to others, and I'm a bit embarrassed to mention it, but I just spent about two hours trying to get a vim-plug installation working on Windows. I've installed it many times before with no issues, but this time, every time I tried to run a command (like "PlugInstall" or "PlugUpgrade"), I'd get the message that the command could not be found.

I eventually tracked it down to the fact that I had recently (temporarily) uninstalled git, which I hadn't realized was an actual requirement. There is the "requirements" page which mentions git, of course, but it's sort of stuck way in the middle where I've apparently managed to overlook it for several years!

Would it be possible to have a link to (or mention of) the requirements closer to the beginning of the installation section, in addition to just way in the middle? It's a very minor change, but it might help people like me on Windows, who accidentally overlook the "requirements" link where it's currently placed....

Just a thought.

------------------------------

```

```

- Type:
    - [ ] Bug
    - [  ] Enhancement
    - [ ] Feature Request
    - [ ] Question
- OS:
    - [ ] All/Other
    - [ ] Linux
    - [ ] OS X
    - [  ] Windows
- Vim:
    - [ ] Terminal Vim
    - [ ] GVim
    - [ ] Neovim

**Comment by junegunn** [comment]
Thanks for the suggestion. But vim-plug will warn you every time you start Vim that it requires `git` executable to be in your `$PATH`, so it's pretty easy to spot the problem early on.

https://github.com/junegunn/vim-plug/blob/cffcfe150bda75177920530c5f1911b52f70a065/plug.vim#L261-L263

![image](https://user-images.githubusercontent.com/700826/116952686-35bccd80-acc6-11eb-889c-d29ceab525fa.png)

You didn't see the message?

**Comment by bratslavia** [comment]
I didn't see anything like that when neovim started, sorry. I had to eventually run ":call plug#begin(~/path/to/stuff)" manually before I saw it.

But looking more closely, that's probably due to some problems in my local installation. If I start in the terminal, the warning you mention shows up just fine, but if I start using the default gui (nvim-qt, which I normally use), it skips past the intro/splash screen almost instantly, and the warning isn't visible. So, probably an issue with my config, or maybe nvim-qt.

Feel free to close the issue!
